console-extensions.json

```
@@ -436,5 +436,25 @@
     "flags": {
       "disallowed": ["MCO"]
     }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "path": "/odf/edit/:cluster/ns/:namespace/:kind/:name/yaml",
```

```suggestion
      "path": ["/odf/edit/:cluster/ns/:namespace/:kind/:name/yaml", "/odf/edit/:cluster/cluster/:kind/:name/yaml"]
```
Remove the other extension

---

For now, we need this change only for DRPolicy, and DRPolicy is cluster scope. so yes, i can remove the other one.